src/prost/build.rs

```diff
@@ -112,6 +112,57 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             "#[derive(::enum_as_inner::EnumAsInner)]",
         )
         .btree_map(btree_map_paths)
+        // node body is a very large enum, so we box it to avoid stack overflow.
+        // TODO: ideally we should box all enum variants automatically https://github.com/tokio-rs/prost/issues/1209
+        .boxed(".stream_plan.StreamNode.node_body.source")
+        .boxed(".stream_plan.StreamNode.node_body.project")
+        .boxed(".stream_plan.StreamNode.node_body.filter")
+        .boxed(".stream_plan.StreamNode.node_body.materialize")
```

Comment:
Should we box the batch plan node as well?

Reply:
batch node body was 680 bytes. After boxing Udf in Expr, it's 464 bytes. Sounds acceptable

Reply:
LocalLookupJoinNode is the largest